lib/dom.js

```diff
@@ -3263,3 +3263,25 @@ declare var status: string;
 declare var top: WindowProxy;
 declare function getSelection(): Selection | null;
 declare var customElements: CustomElementRegistry;
+
+/* Notification */
+declare class Notification extends EventTarget {
+  static permission: string;
+  actions: Array<any>;
```

Comment:
I think this is

    actions: Array<{action: string, title: string, icon?: string}>;